ai-server/app/main.py

```
@@ -8,15 +8,18 @@
 app = FastAPI()
 
 app.include_router(music_router, prefix="/api/v1")
+app.include_router(emotion_router, prefix="/api/v1") 
```

Ruff (0.12.2)

Undefined name `emotion_router`

(F821)